app/forms/video_embed_form_behavior.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-
 # frozen_sting_literal: true
```

Comment:
```suggestion
# frozen_string_literal: true
```

Reply:
i think we just need to delete this line, it's a repeat of line 1 it looks like